lms/djangoapps/courseware/tests/test_password_reset.py

```diff
@@ -1,115 +0,0 @@
-"""
-This file will test through the LMS some of the password reset features
-"""
-from __future__ import absolute_import
-
-from uuid import uuid4
-
-import ddt
-from django.contrib.auth.models import User
-from django.contrib.auth.tokens import default_token_generator
-from django.test.utils import override_settings
-from django.utils.http import int_to_base36
-
-from lms.djangoapps.courseware.tests.helpers import LoginEnrollmentTestCase
-from util.password_policy_validators import create_validator_config
-
-
-@ddt.ddt
-class TestPasswordReset(LoginEnrollmentTestCase):
```

Comment:
Doesn't look like these moved?  Can you explain why they aren't needed?

Reply:
Just to record an in-person conversation, these tests are redundant, which is why they were removed.